lib/context.ts

```diff
@@ -82,7 +82,7 @@ export class Context
 	private _userAgent: string | PerOrigin< string >;
 	private _overwriteUserAgent: boolean | PerOrigin< boolean >;
 	private _accept: string | PerOrigin< string >;
-	private _cookieJar: CookieJar;
+	private _cookieJar: undefined | CookieJar;
```

Comment:
[minor]: since you need to use the same type in another place, so I would suggest that you make this an alias.